stock_available_to_promise_release_carrier_alternative/models/stock_move.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+
+from odoo import models
+from odoo.tools import groupby
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    def _get_estimated_shipping_quantity(self):
+        self.ensure_one()
+        # use available promised qty to estimate the shipping weight
+        return self.ordered_available_to_promise_uom_qty
```

Comment:
I don't see a use for that since the moves are put in a new picking

Reply:
Well, this was the qty that was used in 14 to compute the estimated weight of a move.

Reply:
this is not useful for this module, so please remove it from here